meta/main.yml

```diff
@@ -4,17 +4,17 @@ galaxy_info:
   description: Elastic Repositories
   author: Thomas Widhalm <thomas.widhalm@netways.de>
   company: Netways GmbH
-  license: GPL-3.0-or-later
+  license: license(GPLv3)
```

Comment:
I think this needs to be a valid license ID from https://spdx.org

Reply:
Thank you. Changed it to GPL-3.0-or-later.